pkgs/by-name/ia/ia-writer-mono/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "ia-writer-mono";
+  version = "unstable-2023-06-16";
+
+  src = fetchFromGitHub {
+    owner = "iaolo";
+    repo = "iA-Fonts";
+    rev = "f32c04c3058a75d7ce28919ce70fe8800817491b";
+    hash = "sha256-2T165nFfCzO65/PIHauJA//S+zug5nUwPcg8NUEydfc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/opentype
+    cp "iA Writer Mono/Static/"*.ttf $out/share/fonts/opentype/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "iA Writer Mono Typeface";
+    homepage = "https://ia.net/topics/in-search-of-the-perfect-writing-font";
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = [ ];
+  };
```

```suggestion
  meta = {
    description = "iA Writer Mono Typeface";
    homepage = "https://ia.net/topics/in-search-of-the-perfect-writing-font";
    license = lib.licenses.ofl;
    platforms = lib.platforms.all;
    maintainers = [ ];
  };
```
https://github.com/NixOS/nixpkgs/issues/208242

---

Thanks @langsjo good to know about avoiding `with lib`